components/infobox/wikis/mobilelegends/infobox_unit_hero_custom.lua

```diff
@@ -56,102 +96,45 @@ function CustomUnit:addCustomCells(widgets)
 		Cell{name = 'City', content = {args.city}},
 		Cell{name = 'Attack Type', content = {args.attacktype}},
 		Cell{name = 'Resource Bar', content = {args.resourcebar}},
-		Cell{name = 'Secondary Bar', content = {args.secondarybar}},
-		Cell{name = 'Secondary Attributes', content = {args.secondaryattributes1}},
 		Cell{name = 'Release Date', content = {args.releasedate}},
-		Cell{name = 'Voice Actor(s)', content = CustomUnit._voiceActors(args)}
+		Cell{name = 'Voice Actor(s)', content = CustomHero._voiceActors(args)}
 	)
-
-	local statisticsCells = {
-		hp = {order = 1, name = 'Health'},
-		hpreg = {order = 2, name = 'Health Regen'},
-		mana = {order = 3, name = 'Mana'},
-		manareg = {order = 4, name = 'Mana Regen'},
-		cdr = {order = 5, name = 'Cooldown Reduction'},
-		energy = {order = 6, name = 'Energy'},
-		energyreg = {order = 7, name = 'Energy Regen'},
-		attacktype = {order = 8, name = 'Attack Type'},
-		attackspeed = {order = 9, name = 'Attack Speed'},
-		attackrange = {order = 10, name = 'Attack Range'},
-		damage = {order = 11, name = 'Attack Damage'},
-		ap = {order = 12, name = 'Ability Power'},
-		phyatk = {order = 13, name = 'Physical Damage'},
-		magatk = {order = 14, name = 'Magical Damage'},
-		armor = {order = 15, name = 'Armor'},
-		phydef = {order = 16, name = 'Physical Defense'},
-		magdef = {order = 17, name = 'Magical Defense'},
-		magicresistance = {order = 18, name = 'Magic Resistance'},
-		movespeed = {order = 19, name = 'Movement Speed'},
-	}
-	if Table.any(args, function(key) return statisticsCells[key] end) then
+	
+	if Array.any({'hp', 'hpreg'}, function(key) return String.isNotEmpty(args[key]) end) then
```

Comment:
what if any of the others is not empty but these 2 are?

alternative solution:
```lua
	local baseStats = {
		{name = 'Health', value = args.hp},
		{name = 'Health Regen', value = args.hpreg},
		....
	}

	if Array.any(baseStats, fucntion(item) return Logic.isNotEmpty(item.value) end) then
		table.insert(widgets, Title{name = 'Base Statistics'})
	end

	Array.extendWith(widgets, Array.map(baseStats, function(item)
		return Cell{name = item.name, content = {item.value}}
	end))
```